src/widgets/splits/SplitContainer.cpp

```diff
@@ -849,9 +903,9 @@ void SplitContainer::applyFromDescriptorRecursively(
                 }
                 const auto &splitNode = *inner;
                 auto *split = new Split(this);
+                split->setFilters(splitNode.filters_);
```

Comment:
I changed the order here because we were loading messages before applying the filter when duplicating the tab. I don't think this is a problem, since we call `this->header_->updateChannelText();` in `setChannel` also.